_projects/18-scalac-profiling.md

```diff
@@ -0,0 +1,15 @@
+---
+label: scalacProfiling
+name: Exposing Compilation Performance
+web:
+github: https://github.com/scalacenter/scalac-profiling
+origin: https://github.com/scalacenter/advisoryboard/blob/master/proposals/010-compiler-profiling.md
+contributors: [jvican]
+status: Active
+type: project
+active: true
+hide-from-homepage: false
+description: "Enabling statistics in the compiler and creating the infrastructure around it."
+---
+
+More information is available in the [scalac-profiling](https://github.com/scalacenter/scalac-profiling) repository.
```

Comment:
C'mon. More information than this!

Reply:
Make this easier for someone browsing the Scala Center website to know vaguely what you're doing. We owe it to people!

Reply:
Haha ^^ Alright, I'll add more here.